test/third_party_tests/test_native_executor.py

```diff
@@ -200,6 +200,24 @@ def test_should_run_query_in_mysql(self):
         # Test executions.
         self._execute_native_query()
         self._execute_evadb_query()
+    
+    def test_should_run_query_in_clickhouse(self):
```

Comment:
Does the test needs a clickhouse backend?

Reply:
Yes I will work on it in the upcoming week!

Reply:
@Preethi1609 How do you plan to setup clickhouse? 

@xzdandy @jiashenC Should we add it to CircleCI via docker?

Reply:
Yes we could have a clickhouse server running on 9000?